test/cmd/deployments.sh

```diff
@@ -47,11 +47,7 @@ tryuntil oc get rc/test-deployment-config-1
 # tryuntil oc get rc/test-deployment-config-2
 
 # scale rc via deployment configuration
-oc scale dc test-deployment-config --replicas=1
-oc scale dc test-deployment-config --replicas=2 --timeout=10m
-# scale directly
-oc scale rc test-deployment-config-1 --replicas=4
-oc scale rc test-deployment-config-1 --replicas=5 --timeout=10m
+oc scale dc test-deployment-config --replicas=4
```

Comment:
`./hack/test-cmd` runs only a master so this will never actually scale because of the inability to schedule pods (ie the test is only useful for validating that the command runs). I would suggest adding a stanza in e2e just like what I am doing here: https://github.com/openshift/origin/pull/5852/files